Features/foutafhandeling.feature

```diff
@@ -190,6 +198,89 @@ Abstract Scenario: Ongeldige queryparameter waarde bij zoeken
+
+  Scenario: CRS in bbox-crs parameter niet ondersteund
+    Als een pand wordt geraadpleegd met parameter 'bbox-crs'=b
+    Dan is de http status code in het antwoord 400
+    En is in het antwoord title=CRS b in parameter bbox-crs wordt niet ondersteund.
+    En is in het antwoord status=400
+    En is in het antwoord code=paramCrsNotSupported
+    
+  Scenario: CRS in filter-crs parameter niet ondersteund
+    Als een pand wordt geraadpleegd met parameter 'filter-crs'=b
+    Dan is de http status code in het antwoord 400
+    En is in het antwoord title=CRS b in parameter filter-crs wordt niet ondersteund.
+    En is in het antwoord status=400
+    En is in het antwoord code=paramCrsNotSupported
+
+  Scenario: CRS in crs parameter niet ondersteund
+    Als een pand wordt geraadpleegd met parameter 'crs'=b
+    Dan is de http status code in het antwoord 400
+    En is in het antwoord title=CRS b in parameter crs wordt niet ondersteund.
+    En is in het antwoord status=400
+    En is in het antwoord code=paramCrsNotSupported
+
+  Scenario: CRS in Content-Crs header wordt niet ondersteund
+    Als een pand wordt geraadpleegd met parameter 'Content-Crs'=b
+    Dan is de http status code in het antwoord 400
+    En is in het antwoord title=CRS b in header Content-Crs wordt niet ondersteund.
+    En is in het antwoord status=400
+    En is in het antwoord code=headerCrsNotSupported
+
+  Scenario: bbox-crs parameter niet gespecificeerd
+    Als een pand wordt geraadpleegd met parameter 'bbox'
+    En zonder een parameter 'bbox-crs'
+    Dan is de http status code in het antwoord 400
+    En is in het antwoord title=CRS voor geometrie in parameter bbox-crs moet worden opgegeven.
+    En is in het antwoord status=400
+    En is in het antwoord code=paramCrsNotSpecified
+
+  Scenario: filter-crs parameter niet gespecificeerd
+    Als een pand wordt geraadpleegd met een geometrie anders dan 'bbox', bv. 'point'
+    En zonder een parameter 'filter-crs'
+    Dan is de http status code in het antwoord 400
+    En is in het antwoord title=CRS voor geometrie in parameter filter-crs moet worden opgegeven.
+    En is in het antwoord status=400
+    En is in het antwoord code=paramCrsNotSpecified
+
+  Scenario: crs parameter niet gespecificeerd
+    Als een pand wordt geraadpleegd zonder een parameter 'crs'
+    Dan is de http status code in het antwoord 400
+    En is in het antwoord title=CRS voor resource geometrie moet worden opgegeven.
+    En is in het antwoord status=400
+    En is in het antwoord code=resourceCrsNotSpecified
+
+  Scenario: Content-Crs header niet gespecificeerd
+    Als een pand wordt geraadpleegd met een POST request en een 'point'
+    En zonder een Content-Crs header
+    Dan is de http status code in het antwoord 400
+    En is in het antwoord title=CRS voor geometrie in request body moet worden opgegeven.
+    En is in het antwoord status=400
+    En is in het antwoord code=reqBodyCrsNotSpecified
```

Comment:
Idem als scenario `Scenario: CRS in Content-Crs header wordt niet ondersteund`, dit kan wat mij betreft nu nog niet?